Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -52,11 +52,18 @@ public NodeConnectionParameters Parameters
 			get { return this._Parameters; }
 		}
 
+		private NodeSettings _NodeSettings;
+		public NodeSettings NodeSettings
+		{
+			get { return this._NodeSettings; }
```

Comment:
How about using  public NodeSettings NodeSettings { get; }? it takes quite a lot less place.